src/frontend/src/handler/create_source.rs

```diff
@@ -909,6 +909,8 @@ static CONNECTORS_COMPATIBLE_FORMATS: LazyLock<HashMap<String, HashMap<Format, V
                 POSTGRES_CDC_CONNECTOR => hashmap!(
                     Format::Plain => vec![Encode::Bytes],
                     Format::Debezium => vec![Encode::Json],
+                    // support source stream job
+                    Format::Plain => vec![Encode::Json],
```

Comment:
The `hashmap!` macro looks problematic as it produced no error for the conflict between line 910 and 913.

Btw I am also confused what it really means when `POSTGRES_CDC_CONNECTOR` is used together with `format plain encode bytes`. Seems to be VERY inconsistent with the meaning when `format plain encode bytes` is used with `KAFKA_CONNECTOR` cc @tabVersion @st1page

Reply:
Didn't notice the `format plain encode bytes` combination for direct cdc before. I think we can remove this one, there should not have such use case.

Reply:
Seems introduced by #10592 and agree it looks like unused and removable.

Reply:
agree to remove it for those cdc connectors such as PG, MySQL and CITUS

Reply:
fixed